src/test/java/org/example/automation/amazon/SearchFieldTest.java

```diff
@@ -0,0 +1,87 @@
+package org.example.automation.amazon;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindAll;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class SearchFieldTest extends BaseWebDriverTest {
+    public static final String INCORRECT_INPUT = "890770hhbhb767879zdrjh444:::";
+    public static final String PRESENT = "ipad";
+    public static final String TEXT = "iPad";
+
+    @FindBy(css = ".nav-search-field input")
```

Comment:
It is better to put all elements into special classes of your PageObject. But it is your next task. Hope to see these fields in page classes